includes/Data/SiteFeatures.php

```diff
@@ -105,7 +105,7 @@ public static function get_site_features()
 		);
 	}
 
-	public static function mark_initial_plugins() {
+	private static function mark_initial_plugins() {
```

Comment:
Can we remove lines 4 and 5 from this file since this PR is for removing unused imports?